mantidproject/mantid
# ConvertToReflectometryQ Broken

Fixes #13396. Q-space conversion fixed.

Tester:
- Inspect code changes.
- Check you can reproduce the plot shown in the issue fix. Ask me for data
